tools/compare_pages.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+FILE_TO_COMPARE=$1
+LANGUAGE=$2
+FIRST_PAGE=$3
+
+if [ -z $4 ]; then
+  LAST_PAGE=$3
+else
+  LAST_PAGE=$4
+fi
+
+echo "Making images of ${FILE_TO_COMPARE} and main_${LANGUAGE}.pdf..."
+pdftoppm ${FILE_TO_COMPARE} aa -f ${FIRST_PAGE} -l ${LAST_PAGE} -png -progress &
+sleep 1
```

Comment:
Maybe make it not & and avoid the race condition altogether? Why sleep 1?

Reply:
Without concurrency, it takes twice as long (could be 2 min instead of 1). Sleep 1 is enough to make sure both commands finish before the for loop starts. Actually, in my tests sleep .3 was enough, but I wanted to be on the safe side.

Reply:
Theoretical worst case:
```
tools/compare_pages.sh ~/main_en.pdf en 1 51  221.75s user 2.59s system 238% cpu 1:34.20 total
```
Without concurrency, that's over 3.5 min. With concurrency, it's 1:34.

Reply:
Fixed with `wait`.